README.rst

```diff
@@ -75,6 +61,8 @@ NI-DCPower Python API Status
 +-------------------------------+--------------------------+
 | Supported Python Version      | |nidcpowerPythonVersion| |
 +-------------------------------+--------------------------+
+| Documentation                 | |nidcpowerDocs|          |
```

Comment:
Interesting addition. I didn't realize there was a status badge for documentation. You need to implement this in https://github.com/ni/nimi-python/blob/master/build/templates/status.inc.mako and run `tox -e codegen` with the change.